packages/block-library/src/post-comments-form/style.scss

```diff
@@ -1,5 +1,24 @@
 // Styles copied from button block styles.
```

Comment:
Nit pick: I believe this comment is relevant only for the `input[type="submit"]`?
With this added CSS, it looks out of place, and maybe it can be moved or even removed?

Reply:
And yes it did make me go to the button block style to see where the wildcards were coped from 😄

Reply:
Thanks for spotting that, @carolinan. Relocated 👍